examples/ex25.cpp

```diff
@@ -389,27 +389,21 @@ int main(int argc, char *argv[])
    //     applying any necessary transformations such as: assembly, eliminating
    //     boundary conditions, applying conforming constraints for
    //     non-conforming AMR, etc.
-   a.Assemble();
+   a.Assemble(0);
 
    OperatorHandle Ah;
    Vector B, X;
    a.FormLinearSystem(ess_tdof_list, x, b, Ah, X, B);
 
-   // 13. Transform to monolithic SparseMatrix
-   SparseMatrix *A = Ah.As<ComplexSparseMatrix>()->GetSystemMatrix();
-
-   cout << "Size of linear system: " << A->Height() << endl;
-
-   // 14. Solve using a direct or an iterative solver
+   // 13. Solve using a direct or an iterative solver
 #ifdef MFEM_USE_SUITESPARSE
    {
-      UMFPackSolver  solver(*A);
-      solver.Control[UMFPACK_ORDERING] = UMFPACK_ORDERING_METIS;
-      solver.Mult(B, X);
+      ComplexUMFPackSolver  csolver(*Ah.As<ComplexSparseMatrix>());
+      csolver.Control[UMFPACK_ORDERING] = UMFPACK_ORDERING_METIS;
+      csolver.Mult(B, X);
```

Comment:
I think there should be a some console output, here and in the GMRES section, stating which solver is being used so the user can be sure they're getting what they expect.

Reply:
Added small comment

Reply:
My understanding of the comment from @mlstowell was that the solver itself should output that it is being used. Of course, it should depend on the print level. It is clear when GMRES is being used, since it prints out iterations, but it may be convenient to have `UMFPackSolver` and `ComplexUMFPackSolver` to print out that they are solving, maybe in their `Mult` functions, only if the print level is set for output.

Reply:
UMFPack provides printing options. There is a print_level method in the class to do that. 
0 prints nothing
1 prints only error messages. 
>=2 prints extensive details (flops memory etc) 
For now in ex25 I left it to 1. @mlstowell let me know if this suffices or if you prefer something more elaborate.

Reply:
I should have been more clear.  Just printing the name of the solver choice was fine for me but @dylan-copeland makes a good point.  Turning up the print level to 1 in UMFPack is also a good idea.